developer-process/publish-checklist.md

```diff
@@ -0,0 +1,122 @@
+# Framework publish checklist
+
+
+## Create NPM account
+
+1. Get an account at [npm](https://npmjs.com). Ask framework owners to be added to the proper organizations.
+2. Install [npmrc](https://www.npmjs.com/package/npmrc) and follow its instructions to create a public profile.
+
+
+## Patch release
+
+1. Switch to the latest release branch.
+  ```
+  git checkout <X.x>-release
+  git pull
+  git log
+  ```
+2. Check all the bug/feature tickets under the current github milestone. Make sure the PRs are all cherry-picked to the release branch.
+
+3. If there were several conflicts in cherry-pick process and you have to make several changes in process of selecting partial changes from a PR, create a PR, get teams review, otherwise just push all changes into release branch directly.
+
+4. Test the branch by running `npm run test` and website examples.
+
+5. Update CHANGELOG.md, making sure all commits and PRs merged after release are recorded properly:
+<div align="center">
+  <div>
+    <img src="https://raw.github.com/visgl/deck.gl-data/master/images/dev-docs/publish-guideline/image4.png" />
+    <p><i>Image Text</i></p>
+  </div>
+</div>
+
+6. Make sure you are using the correct NPM profile, then run the publish script:
```

Comment:
Yes, just a note that npmrc can be used to manage multiple configurations should be enough. It will not be an issue for some users, but some contributors in companies that have private npm servers will need to figure this out on their own and that can be a hassle.